LLama.Unittest/Native/SafeLlamaModelHandleVocabularyTests.cs

```diff
@@ -0,0 +1,37 @@
+using System.Text;
+using System.Xml.Linq;
+using LLama.Common;
+using LLama.Extensions;
+using Microsoft.Extensions.Logging;
+
+
+namespace LLama.Unittest.Native;
+
+public class SafeLlamaModelHandleVocabularyTests
+{
+    private readonly LLamaWeights _model;
+
+    public SafeLlamaModelHandleVocabularyTests()
+    {
+        var @params = new ModelParams(Constants.RerankingModelPath)
+        {
+            ContextSize = 0,
+            PoolingType = LLama.Native.LLamaPoolingType.Rank,
+            GpuLayerCount = Constants.CIGpuLayerCount
+        };
+        _model = LLamaWeights.LoadFromFile(@params);
```

Comment:
`_model` is never disposed, should probably make this class disposable (see https://github.com/SciSharp/LLamaSharp/blob/master/LLama.Unittest/BasicTest.cs#L13-L27)